readthedocs/proxito/middleware.py

```diff
@@ -46,8 +46,8 @@ def map_host_to_project_slug(request):  # pylint: disable=too-many-return-statem
     )
 
     # Explicit Project slug being passed in.
-    if 'HTTP_X_RTD_SLUG' in request.META:
-        project_slug = request.META['HTTP_X_RTD_SLUG'].lower()
+    if "HTTP_X_RTD_SLUG" in request.META:
+        project_slug = request.headers["X-Rtd-Slug"].lower()
```

Comment:
Casing doesn't matter, but `Rtd` looks weird p:

```suggestion
        project_slug = request.headers["X-RTD-Slug"].lower()
```

Reply:
[y](url)